http/objectstore/utils/string.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import "time"
+
+//String return points
+func String(v string) *string {
+	return &v
+}
+
+//String return points
+func Time(v time.Time) *time.Time {
+	return &v
+}
```

Comment:
Are these two functions unnecessary and can be used directly with &?

Reply:
Yes, i will delete it later